Assets/MixedRealityToolkit/_Core/Utilities/ClipPlane.cs

```diff
@@ -25,6 +23,7 @@ public class ClipPlane : MonoBehaviour
         private const string clippingPlaneKeyword = "_CLIPPING_PLANE";
         private const string clippingPlaneProperty = "_ClippingPlane";
         private Dictionary<Material, bool> modifiedMaterials = new Dictionary<Material, bool>();
+        private List<Material> allocatedMaterials = new List<Material>();
 
         public Renderer[] Renderers => renderers;
```

Comment:
Do you mind if we co-locate the property and field together?

We also might want to put the tooltip string into a summary tag for the renderers as well.

Reply:
I can do that!